tests/scaletest/scale_test.go

```diff
@@ -1097,3 +1119,21 @@ func TestServiceTypeLB(t *testing.T) {
 	SetupForTesting(t)
 	LBService(t)
 }
+
+func TestUnwantedConfigUpdatesOnAkoReboot(t *testing.T) {
+	SetupForTesting(t)
+	g := gomega.NewGomegaWithT(t)
+	var wg sync.WaitGroup
+	wg.Add(1)
+	// Fetch the controller time before AKO reboot
+	// Use this time as sometimes the time on the controller and test client might be out of sync
+	startTime := FetchControllerTime(t)
+	RebootAko(t, &wg)
+	lib.WaitForAKOPodReboot(t, akoPodName)
+	// Wait for 2 min for AKO to sync
+	time.Sleep(2 * time.Minute)
```

Comment:
These unconditional sleeps add to the overall run time. Can we at least make this sleep conditional? The reason is, AKO may not have synced or may not have booted up during that time frame.